presto-main/src/main/java/com/facebook/presto/server/TaskUpdateRequest.java

```diff
@@ -75,8 +96,9 @@ public SessionRepresentation getSession()
         return extraCredentials;
     }
 
+    @JsonInclude(NON_ABSENT)
```

Comment:
How can this be non-absent? -- the later `TaskUpdateRequest` won't have plan fragment right? -- Am I missing anything? ;)

Reply:
This means don't even include the JSON key if the value is absent